package.json

```diff
@@ -45,24 +44,24 @@
     }
   },
   "dependencies": {
-    "@babel/core": "^7.21.0",
-    "@babel/eslint-parser": "^7.19.1",
```

Comment:
I had mistakenly thought that `@babel/core` needs to be a dependency. By working on [`ember-codemod-remove-ember-css-modules`](https://github.com/ijlee2/embroider-css-modules/blob/0.1.0/ember-codemod-remove-ember-css-modules/package.json#L34-L41), I realized that it can be moved to `devDependencies`.